packages/web/src/public/locales/ar/bible.json

```diff
@@ -0,0 +1,135 @@
+{
+  "reference_format": "{{bookName}} {{chapterNumber}} {{verseNumber}}",
+  "gen": "منشأ",
+  "gen_short": "Gen",
+  "exo": "خروج",
+  "exo_short": "Exo",
+  "lev": "سفر اللاويين",
+  "lev_short": "Lev",
+  "num": "أعداد",
+  "num_short": "Num",
+  "deu": "سفر التثنية",
+  "deu_short": "Deut",
+  "jos": "جوشوا",
+  "jos_short": "Josh",
+  "jdg": "القضاة",
+  "jdg_short": "Judg",
+  "rut": "روث",
+  "rut_short": "Rut",
+  "1sa": "صموئيل الأول 1",
+  "1sa_short": "1 Sam",
+  "2sa": "صموئيل الثاني 2",
+  "2sa_short": "2 Sam",
+  "1ki": "الملوك الأول 1",
+  "1ki_short": "1 Kgs",
+  "2ki": "لملوك الثاني 2",
+  "2ki_short": "2 Kgs",
+  "1ch": "أخبار الأيام الأول 1",
+  "1ch_short": "1 Chr",
+  "2ch": "أخبار الأيام الثاني 2",
```

Comment:
It looks strange, but this is the correct way to do numbers. See [here](https://www.wordproject.org/bibles/ar/index_en.htm#1).